site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -461,6 +461,14 @@ When HTTP POST Bindings are used, this is the default verification key used if a
 +
 This field is required when [field]#Require signature# is enabled.
 
+[field]#Enable login hint# [since]#Available since 1.47.0#::
+When enabled, FusionAuth will accept a username or email address as a login hint on a custom HTTP request parameter
+
+[field]#Login hint parameter name# [optional]#Optional# [default]#defaults to `login_hint`# [since]#Available since 1.47.0#::
+The name of the login hint parameter provided by the service provider on an AuthnRequest. If this parameter is present, its value will be used to pre-populate the username field on the FusionAuth login form.
```

Comment:
I still don't quite get what this does. If I don't, not sure that other folks will. Let me write what I think you are saying:

If I set `application.samlv2Configuration.loginHintConfiguration.parameterName` to `login_name` on an application, then when I set up FusionAuth as an IdP, the SP can send a request which includes the parameter `login_name=richard@example.com` and FusionAuth will pre-populate `richard@example.com` into the login form.

Is that a fair explanation? If so, I'd drop it in here. If not, then please help me understand.

Reply:
Dan, this verbiage ok?